pvlib/tests/test_inverter.py

```diff
@@ -95,6 +95,27 @@ def test_sandia_Pnt_micro():
     assert_series_equal(pacs, pd.Series([-0.043, 132.545914746, 240.0]))
 
 
+def test_sandia_multi(cec_inverter_parameters):
+    vdcs = pd.Series(np.linspace(0, 50, 3))
+    idcs = pd.Series(np.linspace(0, 11, 3)) / 2
```

Comment:
The implementation `test_sandia_multi` appears to allow for a variety of type inputs, so I think it's worth adding regression tests for float and array input. I can imagine several other implementations that would not be so accommodating.